x-pack/legacy/plugins/ml/public/application/components/controls/checkbox_showcharts/index.d.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { BehaviorSubject } from 'rxjs';
+
+export const showCharts$: BehaviorSubject<boolean>;
```

Comment:
Don't think it's a good idea to expose the Subject itself. Would be better to wrap with some service. Also, you might want to expost it `.asObservable()`

Reply:
Thanks! I'm not going to change it in this PR though since it's only a `d.ts` file for existing code that's both used by Anomaly Explorer and Single Metric Viewer. I added a note to the explorer meta issue https://github.com/elastic/kibana/issues/18553